crates/turbopack-node/src/evaluate.rs

```diff
@@ -140,13 +140,32 @@ pub async fn evaluate(
         debug,
     )
     .await?;
-    let mut operation = pool.operation().await?;
+
     let args = args.into_iter().try_join().await?;
-    operation
-        .send(EvalJavaScriptOutgoingMessage::Evaluate {
-            args: args.iter().map(|v| &**v).collect(),
-        })
-        .await?;
+
+    // Sending to workers can fail if reused workers are in a bad state.
+    // To we retry picking workers from the pools until we succeed.
+    let mut i = 1;
+    let mut operation = loop {
+        let mut operation = pool.operation().await?;
+        let result = operation
+            .send(EvalJavaScriptOutgoingMessage::Evaluate {
+                args: args.iter().map(|v| &**v).collect(),
+            })
+            .await;
+        match result {
+            Ok(_) => break operation,
+            Err(err) if i == 10 => {
+                return Err(err);
+            }
+            Err(_) => {
+                // retry
+                i += 1;
+                continue;
+            }
+        }
+    };
```

Comment:
Wrap this in a `retry_future` call. We at least one utility function somewhere for this. Otherwise the meaning of the code is obscured by the retry boilerplate.

Also, put the `10` in a constant somewhere.